doc/user-guide/ICE_tutorial.md

```diff
@@ -123,7 +123,7 @@ If you don't want to compile this application from source, you can just install
 Right after you start [Mangosta-Android] for the first time, you will need to login to your XMPP server.
 In order to do that, just enter the JID you have created for the phone (*phone@myxmpp.com*), the password (*xmpp_password*) and the server address (*2.2.2.2* or *myxmpp.com* if you've set up the domain to actually point to this IP address), and then confirm by clicking "Enter".
 
-<img src="ICE_tutorial/mangosta_login.png" width="25%">
+<img src="mangosta_login.png" width="25%">
```

Comment:
This link was correct.

Reply:
no
https://mongooseim.readthedocs.io/en/latest/user-guide/ICE_tutorial/ICE_tutorial/mangosta_login.png

or add ../ at the beginning, but this is not correct way.

Reply:
just check the page https://mongooseim.readthedocs.io/en/latest/user-guide/ICE_tutorial/ source

Reply:
Yes, it fixes the image for readthedocs but breaks the page on GitHub further. Most probably the ultimate solution here would be to move the .md file to the same directory as the images.

Reply:
well, logo works on github, but next 3 images are broken.
on readthedocs logo is broken but 3 images are working.

maybe it's better to fix for readthedocs first, and then fix completely.